cinema/views.py

```diff
@@ -118,12 +148,23 @@ class OrderPagination(PageNumberPagination):
     max_page_size = 100
 
 
-class OrderViewSet(viewsets.ModelViewSet):
+class OrderViewSet(
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin,
+    viewsets.GenericViewSet,
+):
     queryset = Order.objects.all().prefetch_related(
         "tickets__movie_session__movie", "tickets__movie_session__cinema_hall"
     )
     serializer_class = OrderSerializer
     pagination_class = OrderPagination
+    authentication_classes = (TokenAuthentication,)
+    permission_classes = (IsAuthenticated,)
+
+    def get_permissions(self):
```

Comment:
i don't know if it is a mistake, but this class already inherits from CreateModelMixin and has his own permission class. For me this method seems a bit redundant. If you don't agree just skip my reject, and let's wait what mento says :)

Reply:
agree - seems that it doesn't do much here) thanks!

Reply:
i agree with you - seems like it doesn't do much here) thanks!